# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/gfx-backend-dx11/Cargo.toml
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/gfx-backend-dx11/Cargo.toml b/third_party/rust/gfx-backend-dx11/Cargo.toml
deleted file mode 100644
--- a/third_party/rust/gfx-backend-dx11/Cargo.toml
+++ /dev/null
@@ -1,81 +0,0 @@
-# THIS FILE IS AUTOMATICALLY GENERATED BY CARGO
-#
-# When uploading crates to the registry Cargo will automatically
-# "normalize" Cargo.toml files for maximal compatibility
-# with all versions of Cargo and also rewrite `path` dependencies
-# to registry (e.g., crates.io) dependencies
-#
-# If you believe there's an error in this file please file an
-# issue against the rust-lang/cargo repository. If you're
-# editing this file be aware that the upstream Cargo.toml
-# will likely look very different (and much more reasonable)
-
-[package]
-edition = "2018"
-name = "gfx-backend-dx11"
-version = "0.9.0"
-authors = ["The Gfx-rs Developers"]
-description = "DirectX-11 API backend for gfx-rs"
-homepage = "https://github.com/gfx-rs/gfx"
-documentation = "https://docs.rs/gfx-backend-dx11"
-readme = "README.md"
-keywords = ["graphics", "gamedev"]
-license = "MIT OR Apache-2.0"
-repository = "https://github.com/gfx-rs/gfx"
-[package.metadata.docs.rs]
-default-target = "x86_64-pc-windows-msvc"
-
-[lib]
-name = "gfx_backend_dx11"
-[dependencies.arrayvec]
-version = "0.5"
-
-[dependencies.auxil]
-version = "0.10"
-features = ["spirv_cross"]
-package = "gfx-auxil"
-
-[dependencies.bitflags]
-version = "1"
-
-[dependencies.gfx-renderdoc]
-version = "0.1.0"
-
-[dependencies.hal]
-version = "0.9"
-package = "gfx-hal"
-
-[dependencies.libloading]
-version = "0.7"
-
-[dependencies.log]
-version = "0.4"
-
-[dependencies.parking_lot]
-version = "0.11"
-
-[dependencies.range-alloc]
-version = "0.1"
-
-[dependencies.raw-window-handle]
-version = "0.3"
-
-[dependencies.smallvec]
-version = "1.0"
-
-[dependencies.spirv_cross]
-version = "0.23"
-features = ["hlsl"]
-
-[dependencies.thunderdome]
-version = "0.4"
-
-[dependencies.winapi]
-version = "0.3"
-features = ["basetsd", "d3d11", "d3d11_1", "d3d11sdklayers", "d3dcommon", "d3dcompiler", "dxgi1_2", "dxgi1_3", "dxgi1_4", "dxgi1_5", "dxgiformat", "dxgitype", "handleapi", "minwindef", "synchapi", "unknwnbase", "winbase", "windef", "winerror", "winnt", "winuser"]
-
-[dependencies.wio]
-version = "0.2"
-
-[features]
-default = []